usbmount.conf

```diff
@@ -14,7 +14,7 @@ MOUNTPOINTS="/media/usb0 /media/usb1 /media/usb2 /media/usb3
 
 # Filesystem types: removable storage devices are only mounted if they
 # contain a filesystem type which is in this list.
-FILESYSTEMS="vfat ntfs ext2 ext3 ext4 hfsplus fuseblk ntfs-3g fuse msdos umsdos fat"
+FILESYSTEMS="vfat ntfs ext2 ext3 ext4 hfsplus fuseblk ntfs-3g fuse msdos umsdos fat exfat"
```

Comment:
I think it might be worth adding a follow-on ticket to do an investigation into which file system types our IPCs actually support. For instance, it looks like the `fat` type isn't its own type, but is just a part of the `msdos` type. So, adding it here as an option doesn't actually do anything. I also couldn't find any support for the `umsdos` type, but that doesn't necessarily mean its not there. 

Some of these types were added by my recommendation to try to expand our USB reliability. But, if a file system type was already not supported by the Linux kernel or additional modules, then adding it here won't do anything. It won't harm anything, but it's essentially dead code. All of this to say, anything extra here isn't causing problems, but it might be worth looking into what can be removed.